src/dataProtector/protectDataObservable.ts

```diff
@@ -34,13 +34,14 @@ export const protectDataObservable = ({
   iexec = throwIfMissing(),
   data,
   name = DEFAULT_DATA_NAME,
-  ipfsNodeMultiaddr = DEFAULT_IEXEC_IPFS_NODE,
+  ipfsNode = DEFAULT_IEXEC_IPFS_NODE,
   ipfsGateway = DEFAULT_IPFS_GATEWAY,
 }: IExecConsumer & ProtectDataParams): Observable<ProtectDataMessage> => {
   const vName = stringSchema().label('name').validateSync(name);
-  const vIpfsNodeMultiaddr = stringSchema()
-    .label('ipfsNodeMultiaddr')
-    .validateSync(ipfsNodeMultiaddr);
+  const vIpfsNodeString = stringSchema()
+    .label('ipfsNode')
+    .validateSync(ipfsNode);
```

Comment:
remove this
the string validation is useless because the urlSchema already validates the input as a string

```ts
export const urlSchema = () => string().url('${path} should be a url');
```